edge_cms_mail/owner_conf/views.py

```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import render
+from . import WhoIsWrapper
+from django.core.mail import send_mail
+import smtplib
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+import subprocess
+
+# Views for owner_conf app
+
+@csrf_exempt
+def confirm_owner(request):
+	domain = request.POST.get('domain', '')
+	try:
+		siteInfo = WhoIsWrapper.getSiteInfo(domain)
+	except subprocess.CalledProcessError:
+		# Report failure of whois
+		return JsonResponse({"status": "Failure", "message": "Problem checking domain owner"})
+	except AttributeError:
+		# Report improper output of whois
+		return JsonResponse({"status": "Failure", "message": "Unable to find owner for {}".format(domain)})
+
+	# Obtain owner info and send email
+	owner = siteInfo.name
+	email = siteInfo.email
+	try:
+		send_mail(
+			"Please Confirm Email Address",
+			'''Hello {},
+				Thank you for using edge-cms on your domain: {}'''.format(owner, domain),
+			"DoNotReply@edge-cms.com",
+			[email],
+			fail_silently=True)
```

Comment:
Could you expand on why you have fail_silently = True as well as a check for smtplib.SMTPException?